spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/ConvLSTMPeephole.scala

```diff
@@ -54,6 +56,8 @@ class ConvLSTMPeephole[T : ClassTag](
   val kernelC: Int,
   val stride: Int = 1,
   val padding: Int = -1,
+  var activation: TensorModule[T] = null,
```

Comment:
Looks like Keras defines activation as String and create an instance when using it. Maybe we can have the same behavior, one benefit is we don't need clone it every time.